src/components/Button.jsx

```diff
@@ -1,44 +1,67 @@
-import * as React from 'react';
-import { ViewPropTypes, StyleSheet, Text, TouchableHighlight } from 'react-native';
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  ViewPropTypes,
+  StyleSheet,
+  Text,
+  TouchableHighlight,
+} from 'react-native'
 
 const styles = StyleSheet.create({
   touchable: {
-    flexDirection: "row", flexWrap: "wrap", justifyContent: "center", alignItems: "center"
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    alignItems: 'center',
   },
   touchableMain: {
-    backgroundColor: '#4360DF1a', borderRadius: 8
+    backgroundColor: '#4360DF1a',
```

Comment:
Can you use the proper color from palette here? Just to see how it would be done :)

Reply:
Sure. Good point. I'll update this one 👍